packages/deepbook/sources/pool.move

```diff
@@ -671,6 +681,48 @@ public fun update_allowed_versions<BaseAsset, QuoteAsset>(
     inner.allowed_versions = allowed_versions;
 }
 
+/// Adjust the tick size of the pool. Only admin can adjust the tick size.
+public fun adjust_tick_size_admin<BaseAsset, QuoteAsset>(
+    self: &mut Pool<BaseAsset, QuoteAsset>,
+    new_tick_size: u64,
+    _cap: &DeepbookAdminCap,
+) {
+    let self = self.load_inner_mut();
+    assert!(new_tick_size > 0, EInvalidTickSize);
```

Comment:
can we make more asserts here? new_tick_size must be strictly less than tick_size and must be a power of 10

Reply:
tick size can be increased with no problem right? won't be used other than when the order is initially placed (and only admin can do this, more restrictions on public functions after permissionless upgrade)

Reply:
I guess it can. We should still ensure they are always powers of 10